apps/package.json

```diff
@@ -15,6 +15,7 @@
     "test:integration": "grunt integrationTest",
     "test:entry": "grunt karma:entry",
     "test:code-studio": "grunt codeStudioTest",
+    "flow": "flow src",
```

Comment:
I highly recommend enabling it for tests. Maybe not now, but eventually. One of the huge benefits of a nearly-completely flowified codebase is that it catches all consumers when you refactor things.